include/gazebo_ros_link_attacher/gazebo_ros_link_attacher.h

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2024, SENAI Cimatec
```

Comment:
The `uncrustify` check conflicts with `cpplint` about this file. It's not a critical error, but it is the root of the error on the tests with colcon. Tests with autoproj are passing completely, though. One can check it by:
Formatting with uncrustify:
```bash
ament_uncrustify --reformat include/gazebo_ros_link_attacher/gazebo_ros_link_attacher.h
```
Check with cpplint:
```bash
ament_cpplint
```

Reply:
done in https://github.com/Brazilian-Institute-of-Robotics/gazebo_ros_link_attacher/commit/d1afd8f98b012a6dadaa6aefd431bcd1c6dbcefd